plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/ApplicationGIS.java

```diff
@@ -976,9 +988,9 @@ public int getDPI() {
      * @param activeMapTracker the tracker that managers
      */
     public static void setActiveMapTracker( ActiveMapTracker activeMapTrackerToSet ) {
-        if (activeMapTracker != null ){
+        /*if (activeMapTracker != null ){
```

Comment:
What's the reason behind to comment this block? It there a usecase to allow to re-set map-tracker?